applications/psd_pipeline/data_writer/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "operator": {
-        "name": "data_writer",
+        "name": "Data Writer Operator",
```

Comment:
whats wrong using this name? I believe it's the same as the cmake target name which makes it convenient to recognize and use quickly as that defines the name in the holohub website navigation.

Reply:
most operators have that format: https://nvidia-holoscan.github.io/holohub/operators/

Reply:
@agirault I agree, It is fine to have the names like that but we should be consistent across different files. I will revert this as it is not in the scope of this PR but let's discuss about the operator naming online.